Testscripts/Windows/PERF-SCHEDULER.ps1

```diff
@@ -0,0 +1,208 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+param(
+    [String] $TestParams,
+    [object] $AllVMData,
+    [object] $CurrentTestData
+)
+
+function Main {
+    param (
+        $TestParams,
+        $AllVMData
+    )
+    # Create test result
+    $CurrentTestResult = Create-TestResultObject
+    $resultArr = @()
+
+    try {
+        $testName="scheduler"
+        $myString = @"
+./perf_scheduler.sh &> schedulerConsoleLogs.txt
+. utils.sh
+collect_VM_properties
+"@
+        Set-Content "$LogDir\Start${testName}Test.sh" $myString
+        Copy-RemoteFiles -uploadTo $AllVMData.PublicIP -port $AllVMData.SSHPort `
+            -files "$constantsFile,$LogDir\Start${testName}Test.sh" -username $user -password $password -upload
+        Copy-RemoteFiles -uploadTo $AllVMData.PublicIP -port $AllVMData.SSHPort `
+            -files $currentTestData.files -username $user -password $password -upload
+
+        Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password `
```

Comment:
@johnsongeorge-w recommended me to use "bash \<bash script\>" without file mode change. Try it!

Reply:
I had tried. But I also need to "chmod utils.sh", or there will be "Permission denied" error.

Reply:
Can you please give more details. I think utils.sh is sourced in test script and there should not be any issue when running using bash.